lib/modules/manager/npm/update/dependency/index.ts

```diff
@@ -30,6 +31,18 @@ function renameObjKey(
   }, {} as DependenciesMeta);
 }
 
+function getStrValue(isYaml: boolean, text: string): string {
+  if (!isYaml) {
+    return `"${text}"`;
+  }
+  const simpleString = new RegExp(/^[a-z0-9]/i).exec(text);
```

Comment:
```suggestion
  const simpleString = new RegExp(/^[a-z0-9]/i).exec(text);
```

Use `regEx()` instead

Reply:
So `foo` or `1` or `foo1` are simple strings, but `1.2` would not be?

Reply:
no, I only check that starts with any letter or number.